composer.json

```
@@ -5,7 +5,7 @@
   "keywords": ["TYPO3 CMS", "CLI", "Cache"],
   "license": "GPL-2.0-or-later",
   "require": {
-    "typo3/cms-core": "^8.7.0 || ^9.5 || ^10.0"
+    "typo3/cms-core": "^8.7.0 || ^9.5 || ^10.0 || ^11.0"
```

This is no longer correct, is it?
The checks for Versions 8 and 9 have been removed in the code.
```suggestion
    "typo3/cms-core": "^10.0 || ^11.0"
```